pkg/config.go

```diff
@@ -11,12 +13,14 @@ import (
 type Config struct {
 	ReIterationTime time.Duration `yaml:"re_iteration_time"`
 	LogLevel        string        `yaml:"log_level"`
+	Secret          string        `yaml:"secret" type:"secret"`
```

Comment:
change to
```
DefaultKey          string        `yaml:"default_key" type:"secret"`
```

Reply:
@RaiBnod and I decided to use `Secret` for a more generic approach. This also aligns with how the secret values are shown in RCE. Is it okay if we use this property?

Reply:
Let's add `default_key` as a secret in the frontend as well and also change the key here.

Reply:
Okay 👍